tbx/project_styleguide/templates/patterns/molecules/blog-item/blog-item.html

```diff
@@ -18,8 +18,21 @@ <h2>{{ item.title }}</h2>
         {% if item.event_type %}
             {{ item.event_type }}
         {% endif %}
-        {% if item.date %}
+        {% if item.date and item.type != "BLOG POST" %}
```

Comment:
Query: what does `item.type` do here? would all the posts not be blog posts?

Reply:
I am completely updating this template in the blog listing FE so it would be simpler to leave it un-edited.

Reply:
> Query: what does `item.type` do here? would all the posts not be blog posts?

Good question! I think I missed the fact that this template wasn't used by anything other than blog posts. Removed in feefc30